dags/search_forecasting.py

```diff
@@ -0,0 +1,59 @@
+"""
+See [kpi-forecasting in the docker-etl repository](https://github.com/mozilla/docker-etl/blob/main/jobs/kpi-forecasting).
+
+This DAG runs the search forecasts for the DAU, search count and ad clicks metrics .
+
+This DAG is high priority for week 1 of the month and low priority otherwise.
+"""
+
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.tags import Tag
+
+default_args = {
+    "owner": "jsnyder@mozilla.com",
+    "email": ["jsnyder@mozilla.com", "mbowerman@mozilla.com"],
+    "depends_on_past": False,
+    "start_date": datetime(2024, 6, 21),
+    "email_on_failure": True,
+    "email_on_retry": True,
```

Comment:
```suggestion
    "email_on_retry": False,
```

Generally, we do not email on retry. The only valid use case for keeping this enabled would be if you actually expect someone to respond to this try failing immediately otherwise it's just noise.

Reply:
Whoops, fixed in latest commit